app/models/tslr_claim.rb

```diff
@@ -120,6 +120,14 @@ def student_loan_country_with_one_plan?
     StudentLoans::PLAN_1_COUNTRIES.include?(student_loan_country)
   end
 
+  def address_verified?
+    (ADDRESS_ATTRIBUTES & verified_fields).any?
```

Comment:
New syntax to me. What does this do? I'm assuming from context that `&` gets the intersection of the two arrays?

Reply:
Yep, binary ampersand performs set intersection in Ruby when used with arrays.